services/api/src/api/routes/croissant.py

```diff
@@ -122,38 +126,47 @@ def get_croissant_from_dataset_infos(
             fields_names: set[str] = set()
             if isinstance(feature, Value) and feature.dtype in HF_TO_CROISSANT_VALUE_TYPE:
                 fields.append(
-                    {
-                        "@type": "ml:Field",
-                        "name": _escape_name(column, fields_names),
-                        "description": f"Column '{column}' from the Hugging Face parquet file.",
-                        "dataType": HF_TO_CROISSANT_VALUE_TYPE[feature.dtype],
-                        "source": {"distribution": distribution_name, "extract": {"column": column}},
-                    }
+                    _remove_none_values(
+                        {
+                            "@type": "ml:Field",
+                            "@id": _escape_name(column, fields_names) if croissant_1_0 else None,
+                            "name": _escape_name(column, fields_names),
+                            "description": f"Column '{column}' from the Hugging Face parquet file.",
+                            "dataType": HF_TO_CROISSANT_VALUE_TYPE[feature.dtype],
+                            "source": {"fileSet": {"@id": distribution_name}, "extract": {"column": column}},
```

Comment:
Should you distinguish v0.8 and v1.0 using the flag?

```
"source": {"distribution": distribution_name, "extract": {"column": column}},

"source": {"fileSet": {"@id": distribution_name}, "extract": {"column": column}},
```

Reply:
Totally, I forgot that!